nupic/research/frameworks/pytorch/imagenet/imagenet_experiment.py

```diff
@@ -422,22 +409,10 @@ def post_batch(self, model, loss, batch_idx, epoch, num_images, time_string):
             self.logger.debug("Timing: %s", time_string)
 
     def post_epoch(self, epoch):
-        count_nnz = self.logger.isEnabledFor(logging.DEBUG) and self.rank == 0
-        if count_nnz:
-            params_sparse, nonzero_params_sparse1 = count_nonzero_params(self.model)
-
-        self.model.apply(rezero_weights)
         if self.post_epoch_hooks:
```

Comment:
Do we still need post_epoch_hooks or should these just be done in mixins?

Reply:
Cool, I went ahead and removed the `post_epoch_hooks` and `init_hook`. In the future we could also move `init_batch_norm` into a mixin.